docs/executor-custom.md

```diff
@@ -101,11 +162,209 @@ echo "http://google.pl" | kubectl testkube tests create --name example-google-te
+As for now testkube runs directly command and is passing execution information as argument 
+We need to add runner binary (but we have plans to remove need of this step)
+
+```sh
+#!/usr/bin/env sh
+node app.js "$@"
+```
+
+And add it into our Dockerfile
+
+```Dockerfile
+FROM node:17
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Bundle app source
+COPY runner /bin/runner
+RUN chmod +x /bin/runner
+COPY app.js app.js
+
+EXPOSE 8080
+CMD [ "/bin/runner" ]
+```
+
+Now let's build and push our docker container (change user/repo to your Docker Hub username): 
+```sh
+docker build --platform=linux/amd64 -t USER/testkube-executor-example-nodejs:latest -f Dockerfile .
+docker push USER/testkube-executor-example-nodejs:latest
+```
+
+After we have our image in place where Kubnernetes can load it we need to define our executor: 
+
+```yaml
+apiVersion: executor.testkube.io/v1
+kind: Executor
+metadata:
+  name: example-nodejs-executor
+  namespace: testkube
+spec:
+  executor_type: job
+  features: []
+  image: kubeshop/testkube-executor-example-nodejs:latest
+  types:
+    - example/test
+```
+
+Save it somewhere e.g. `example-executor.yaml` and apply into Kubernetes cluster: 
+
+```sh
+kubectl apply -f example-executor.yaml
+```
+
+
+When everything in place we can now start adding our Testkube tests 
+(We'll need testkube for this so head to [installation instructions](/testkube/installing/))
+
+```
+echo "https://httpstat.us/200" | kubectl testkube tests create --name example-test --type example/test
+```
+
+As we can see we need to pass test name and test type (`example/test` which we defined in our executor CRD). 
+
+Now it's finally time to run our test!
+
+```
+kubectl tests run example-test -f
+
+████████ ███████ ███████ ████████ ██   ██ ██    ██ ██████  ███████ 
+   ██    ██      ██         ██    ██  ██  ██    ██ ██   ██ ██      
+   ██    █████   ███████    ██    █████   ██    ██ ██████  █████   
+   ██    ██           ██    ██    ██  ██  ██    ██ ██   ██ ██      
+   ██    ███████ ███████    ██    ██   ██  ██████  ██████  ███████ 
+                                           /tɛst kjub/ by Kubeshop
+
+
+Type          : example/test
+Name          : example-test-string
+Execution ID  : 6218ccd2a26fa94ee7a7cfd1
+Execution name: moderately-pleasant-labrador
+
+
+Getting pod logs
+Execution completed Got valid status code: 200 OK
+
+.
+Use following command to get test execution details:
+$ kubectl testkube tests execution 6218ccd2a26fa94ee7a7cfd1
+
+
+
+Got valid status code: 200 OK
+Test execution completed with sucess in 6.163s 🥇
+
+Use following command to get test execution details:
+$ kubectl testkube tests execution 6218ccd2a26fa94ee7a7cfd1
+
+
+Watch test execution until complete:
+$ kubectl testkube tests watch 6218ccd2a26fa94ee7a7cfd1
+
+
+```
+
+Yay our test completes with success!
+(you can try to create another test with diferent status code and check how it's failing)
```

Comment:
Create another test with a different status code and check to see how it's failing.